Outset/Outset.swift

```diff
@@ -309,7 +309,7 @@ struct Outset: ParsableCommand {
             ensureRoot("add scripts to override list")
 
             for var override in addOverride {
-                if !override.contains(loginOnceDir) {
+                if !override.contains(loginOnceDir) && !override.contains(loginOncePrivilegedDir) {
```

Comment:
that should work - the idea here is you can pass just the script name and with no full path it will make an assumption about it being a login-once.

Might me better to make it less reliant on a specific path (e.g. if another `once` folder is added in the future then this will need to change again)

maybe check to see if the passed in override has a path component and if not, then assume `login-once`